config/database.yml

```
@@ -13,9 +13,10 @@ default: &default
   adapter: mysql2
   encoding: utf8mb4
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  username: root
-  password:
-  host: localhost
+  username: <%= ENV['MYSQL_ROOT_USER'] %>
+  password: <%= ENV['MYSQL_ROOT_PASSWORD'] %>
+  host: <%= ENV['MYSQL_HOST'] %>
+  port: <%= ENV['MYSQL_PORT'] %>
```

(すいません五月雨になってしまいました、、)
こうしてみると下記の点が気になりました。

- DBはMYSQLとは限らないのにMYSQLが名前に入っている
- rootユーザーとは限らないのにROOTが入っている

```suggestion
  username: <%= ENV['DB_USER'] %>
  password: <%= ENV['DB_PASSWORD'] %>
  host: <%= ENV['DB_HOST'] %>
  port: <%= ENV['DB_PORT'] %>
```

ちなみにrootは権限強すぎるので基本的に使っちゃだめなのですが、開発環境ではrootでやってしまうことも多いです。(非rootユーザーを作るのが面倒なので。mysqlの公式dockerイメージは環境変数を渡すことで非rootユーザーとDBを一つ作ることもできますが、esaの例だとrootしか無いです。さらに下の方の developmentやtestの設定も変えないといけなくなってきますが、今回はrootのuser/passを入れておけばいいと思います〜)

---

@a-hirakawa 
ご指摘いただいた件を修正して再プッシュしました！
言われてみればその通りですね…。
改めましてご確認をお願いします。